pkg/helm/wrapper.go

```diff
@@ -13,10 +13,13 @@ import (
 var (
 	TillerNamespace      = "kube-system"
 	TillerServiceAccount = "tiller"
+	RancherNamespace     = "cattle-system"
+	RancherSecret        = "tls-ca"
+	RancherCAFilename    = "cacerts.pem"
 )
 
 type Helm interface {
-	Init() error
+	Init(cacert []byte) error
```

Comment:
Thoughts:

The primary concern of the `helm` package is wrapping Helm operations. I was planning on breaking this out into its own proper library because we could already use it inside Shipyard, and I'm almost positive we'll need some Helm Go support in a future project. From a responsibility standpoint, this really belongs inside the `ranchhand` pkg.

1. We can easily create a k8s config/clientset inside the `ranchhand` pkg and then add the creation of the CA PEM somewhere in the sequence of operations; `installRancher` seems to make the most sense to me.
2. If we don't actually need to create a k8s secret for this CA PEM, then this change can go away completely. [This doc](https://rancher.com/docs/rancher/v2.x/en/installation/ha/helm-rancher/tls-secrets/) appears to indicate that this is only necessary when using Rancher to manage ingress certificates.

Reply:
We did discuss that adding the cert into k8s was necessary, though I'm not sure we got into the specific implementation details. I piggybacked on createK8sResources because it's the only place we're talking directly to k8s at present, but you're right that it isn't a helm concern (and wouldn't make sense if it were broken out into another library). I can move the k8s operations from that function into a function in pkg/rancher.go that we call just before helm, if that's the structure you're looking for, though I think you said something about using helm natively.

LMK the direction you'd like to take.